cockpit/util/datadoc.py

```diff
@@ -687,6 +687,7 @@ def writeMrcHeader(header, filehandle):
 # dimensions it will be augmented with dimensions of size 1 starting from
 # the left (e.g. a 512x512 array becomes a 1x1x1x512x512 array).
 def writeDataAsMrc(data, filename, XYSize = None, ZSize = None, wavelengths = []):
+    data = data.copy() # Do not modify original data.
```

Comment:
This makes an entire copy of the data to avoid modifying the original data. However, it seems that the changes are only done to the array shape. Instead of making a whole copy, we can use reshape to get a reference to the same data where only the shape is modified.

Reply:
I thought about that, but was concerned a non-contiguous view might cause problems with the mem-mapped file access used by Mrc. I'll try it.

Reply:
Using a view seems to work. Resolved in 35493993136ac96741299111c330474bb76068f3